build/VERSIONS

```diff
@@ -0,0 +1,4 @@
+ZLIB_VERSION=1.2.11
+LIBEVENT_VERSION=2.1.12-stable
+OPENSSL_VERSION=1.1.1h
+LIBTOR_VERSION=0.3.5.11-dev
```

Comment:
Wouldn't be possible to wire this to `lock.json` to follow the build script ?

Reply:
Yes it would be a good idea, I mentioned it in the description, but I don't have time to do that in this PR.